app/assets/stylesheets/application.scss

```diff
@@ -52,6 +53,8 @@ $covid-yellow: #fff500;
 @import "views/taxons";
 @import "views/organisations";
 @import "views/organisation";
+@import "views/postcode-lookup";
```

Comment:
Should we move this into `shared/postcode-lookup` to keep it consistent with the template

Reply:
thanks for moving!